scripts/reana

```diff
@@ -0,0 +1,112 @@
+# submit itself to any jurisdiction.
+
+reana_components="reana-job-controller reana-workflow-controller
+                  reana-workflow-monitor reana-message-broker
+                  reana-workflow-engine-yadage"
+
+install_local_minikube () {
+    # Check whether the OS is supported
+    if ! echo "darwin linux" | grep -F -q -w -i "$(uname -s)"
+    then
+        echo 'Your OS is not supported.'
+        exit 1
+    else
+        OS="$(uname -s | tr '[:upper:]' '[:lower:]')"
+    fi
+
+    # Install tools `kubectl` and `minikube` if necessary
+    if ! command -v kubectl > /dev/null 2>&1
+    then
+        echo 'Installing latest kubectl ...'
+        curl -LO "https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/$OS/amd64/kubectl"
+        chmod +x ./kubectl
+        sudo mv ./kubectl /usr/local/bin/kubectl
+    fi
+
+    if ! command -v minikube > /dev/null 2>&1
+    then
+        echo 'Installing minikube ...'
+        curl -Lo minikube "https://storage.googleapis.com/minikube/releases/v0.17.1/minikube-$OS-amd64"
+        chmod +x minikube
+        sudo mv minikube /usr/local/bin/
+    fi
+
+    # FIXME find better way to define this dependency.
+    # `extras_require` does not support not PyPI deps and
+    # `dependency_links` does not seem to work.
+    pip install -r requirements-k8s.txt
+
+    # Start `minikube`
+    minikube start || echo 'Error while starting minikube' && exit 1
+
+    # Use minikube Docker daemon, so local images will be used instead
+    # of pulling from registry
+    echo 'Using minikube Docker daemon ...'
+    eval "$(minikube docker-env)"
+
+    # Pull all REANA components images and retag for local `dev`
+    echo 'Pulling REANA components images ...'
+    for component in $reana_components
+    do
+        docker pull "reanahub/$component"
+        docker tag "reanahub/$component" "$component"
+    done
+
+    # Generate Kubernetes manifests
+    echo 'Building REANA configuration manifests ...'
+    reana-resources-k8s build-manifests
+
+    # Set current cluster secret for `reana-job-controller` component
+    current_cluster_secret=$(kubectl get secrets | grep 'service-account-token' | cut -d ' ' -f 1)
+    sed -i'' -e "s/default-token-8p260/$current_cluster_secret/" configuration-manifests/deployments/job-controller.yaml
+
+    # Deploy system
+    echo 'Deploying REANA on minikube ...'
+    kubectl create -Rf configuration-manifests
+
+    # Clean configuration manifests
+    rm -rf configuration-manifests
+}
+
+usage() {
+    echo "${0##*/} manages REANA system operations"
+    echo ""
+    echo "Usage:"
+    echo "  install-minikube    Deploys local minikube based REANA cluster"
+    echo "  -h                  Help"
+}
+
+if [ "$1" != "" ]
+then
+    case $1 in
+        install-minikube ) install_local_minikube
+                           ;;
+        clean-minikube)    minikube delete
```

Comment:
Since our discussion last week, we decided to leave backend specific management up to the user. Just a minimal command for installing and running `Kubernetes` is provided. However, if it doesn't work (he have to investigate your problem to see if it is easy to fix) I would suggest to remove it from the CLI and stick to REANA specific operations.